web_widget_x2many_2d_matrix/static/src/js/web_widget_x2many_2d_matrix.js

```diff
@@ -261,6 +254,26 @@ openerp.web_widget_x2many_2d_matrix = function(instance)
             return this.by_x_axis[x][y]['id'];
         },
 
+        get_xy_att: function(x, y)
+        {
+            vals = {};
```

Comment:
in js, you should declare variables with the var keyword. Otherwise, you'll pollute the global namespace which can lead to weird and hard to debug interactions